### PR TITLE
fix post code

### PR DESCRIPTION
**Pull request by diogoteix**

### CODE REVIEW

File: Trackovid19-web/src/app/screens/post-code/post-code.component.html [hunk 1]
```diff
@@ -24,13 +24,9 @@
             </div>
 
             <div class="policy-terms">
-              <a class="text-terms text-grey text-center" target="_blank" href="/terms-conditions"
```
Comment (fabioscsilva) [inline Trackovid19-web/src/app/screens/post-code/post-code.component.html:27]: Is to remove the target _blank?

Comment (fabioscsilva) [reply Trackovid19-web/src/app/screens/post-code/post-code.component.html:27]: Forget, on this this we shouldn't have url at all

File: Trackovid19-web/src/app/screens/post-code/post-code.component.html [hunk 2]
```diff
@@ -85,7 +81,7 @@
         <input
           class="second-type-number"
           formControlName="zip-code-2"
-          type="text"
+          type="number"
           pattern="[0-9]*"
```
Comment (fabioscsilva) [inline Trackovid19-web/src/app/screens/post-code/post-code.component.html]: I believe that we should block to 3 digits here, can you do it?

File: Trackovid19-web/src/app/screens/post-code/post-code.component.html [hunk 3]
```diff
@@ -75,7 +71,7 @@
         <input
           class="frist-type-number"
           formControlName="zip-code-1"
-          type="text"
+          type="number"
           pattern="[0-9]*"
```
Comment (fabioscsilva) [inline Trackovid19-web/src/app/screens/post-code/post-code.component.html]: I believe that we should block to 4 digits here, can you do it?

Comment (fabioscsilva) [inline Trackovid19-web/src/app/screens/post-code/post-code.component.html]: Can you add a validation to have 4 number? It should be possible to send 3 and now we can do it

Comment (diogoteix) [reply Trackovid19-web/src/app/screens/post-code/post-code.component.html]: Done

File: Trackovid19-web/src/app/screens/post-code/post-code.component.html [hunk 4]
```diff
@@ -85,8 +82,8 @@
         <input
           class="second-type-number"
           formControlName="zip-code-2"
-          type="text"
-          pattern="[0-9]*"
+          type="number"
```
Comment (bmm) [inline Trackovid19-web/src/app/screens/post-code/post-code.component.html]: O type foi alterado para text porque quando começava com 0, o 0 era ignorado

Comment (diogoteix) [reply Trackovid19-web/src/app/screens/post-code/post-code.component.html]: Done!

**Review by bmm — APPROVED** [review]

**Pull request merged**